data-reconciliation-app/src/endpoints/reporting.ts

```diff
@@ -0,0 +1,31 @@
+import * as ccfapp from "@microsoft/ccf-app";
+import { ServiceResult } from "../utils/service-result";
+import { reportingService, authenticationService } from "../utils/dependencies";
+import { ApiResult } from "../utils/api-result";
+
+export function getDataSummary(
+  request: ccfapp.Request<any>
```

Comment:
We can update `prettier line width` to 240. We don't have to write the method declaration in this format 🐱 
```suggestion
  export function getDataSummary(request: ccfapp.Request<any>): ccfapp.Response<any>
```

Reply:
The response type can be `ccfapp.Response<T>` where T can be an interface returned from `reportingService.getData`.

```typescript
export interface ReportingResponse {
 //////
}